src/Microsoft.NET.Sdk.Functions.MSBuild/Tasks/Http/DefaultHttpClient.cs

```diff
@@ -1,13 +1,17 @@
 ﻿using System;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.NET.Sdk.Functions.Http
 {
     internal class DefaultHttpClient : IHttpClient, IDisposable
     {
-        private HttpClient _httpClient = new HttpClient();
+        private readonly HttpClient _httpClient = new HttpClient()
```

Comment:
Not particular to your changes, but should we pass a clientRequestHandler into the HttpClient constructor that sets the accept-encoding to gzip;deflate for better compression (e.g performance) on the responses?  Are they big enough that it is interesting?  With Fiddler, you can switch the compression of the responses and see the different sizes that result.

Reply:
The use case of the client in this case is minimal. We can certainly revisit this if the client scenarios gets more advanced..

Reply:
Tracking : https://github.com/Azure/azure-functions-vs-build-sdk/issues/461